### PR TITLE
SC-5806 first login 

### PR DESCRIPTION
**Pull request by luizzaly**
# Description
Improved a11y of  first login 

## Links to Tickets or other pull requests

Base links to copy

- https://ticketsystem.schul-cloud.org/browse/SC-5806

## Changes

aria labels
alt 
translation

### CODE REVIEW

File: theme/open/views/registration/forms/einwilligungserklaerung.hbs [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
+	<p class="explanation">Die Einwilligung und damit die Nutzung ist freiwillig, das heißt, wenn du deine Einwilligung nicht erteilst, führt diese für dich nicht zu Nachteilen im Unterricht. Eine Teilnahme am Unterricht, außerhalb der {{@root.theme.title}}, wird durch die Schule ermöglicht werden. Die Einwilligung kann jederzeit von dir widerrufen werden. Eine kurze E-Mail mit der Erklärung des Widerrufes, der Angabe des Vor- und Nachnamens, der Klasse und der Schule an folgende Adresse genügt: <a href="mailto:widerruf@schul-cloud.org?subject=Widerruf der Einwilligung" target="_blank">widerruf@schul-cloud.org</a>. Im Fall des Widerrufs löschen wir die auf Basis der Einwilligung gespeicherten personenbezogenen Daten unverzüglich, es sei denn, es besteht ein gesetzlicher Grund zur Aufbewahrung oder du willst vorher noch auf die gespeicherten Daten zugreifen. Weitere Hinweise zum Datenschutz kannst du auch unseren <a class="nav-link" target="_blank" href="https://docs.schul-cloud.org/x/GxfGBg">FAQ</a> entnehmen.</p>
```
Comment (haniapalac) [inline theme/open/views/registration/forms/einwilligungserklaerung.hbs:24]: If it is a quick change maybe add a warning about the link opening in the new tab.

File: views/firstLogin/sections/thanks.hbs [hunk 2]
```diff
@@ -2,14 +2,14 @@
     <div class="welcomeGreeting">
         <div class="welcomeGreetingImage text-center" id="welcome">
             <div class="content">
-                <img src="/images/firstLogin/welcome_screen.svg">
+                <img alt="{{$t "first_login.img_alt.wavingStudents"}}" src="/images/firstLogin/welcome_screen.svg">
```
Comment (haniapalac) [inline views/firstLogin/sections/thanks.hbs:5]: Isn't the image for presentational purpose only? then I would change the role and hide it from AT.

File: views/firstLogin/sections/thanks.hbs [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
                 <hr class="mt-0">
                     <h1 class="h2">{{$t "firstLogin.welcome.text.welcomeTo" (dict "shortTitle" @root.theme.short_title)}}</h1>
                 <hr>
                 <!-- fokus here -->
-                <a href=""></a> 
+                <a aria-hidden="true" href=""></a>
```
Comment (haniapalac) [inline views/firstLogin/sections/thanks.hbs:10]: Why is the link hidden?

File: views/firstLogin/sections/thanks.hbs [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
                 <div class="welcomeGreetingScroll">
-                    <div class="btn-scroll text-center mt-2">
+                    <div aria-hidden="true" class="btn-scroll text-center mt-2">
```
Comment (haniapalac) [inline views/firstLogin/sections/thanks.hbs:12]: I guess a button would be better than a div.

File: theme/open/views/registration/forms/einwilligungserklaerung.hbs (hunk 1 above)
Comment (ABialoskorski) [inline theme/open/views/registration/forms/einwilligungserklaerung.hbs:24]: why this is not in de.json?

Comment (luizzaly) [reply theme/open/views/registration/forms/einwilligungserklaerung.hbs:24]: I guess aria is not useful as the screen reader will only announce the "opens in a new tab" and via css would be quite some code I guess, but maybe we should decide how we want to proceed with links or documents opening in a new tab....🤔

File: views/firstLogin/sections/thanks.hbs (hunk 3 above)
Comment (luizzaly) [reply views/firstLogin/sections/thanks.hbs:10]: I got rid of the empty a tag and I think the arrow down is a nice to have but not necessary for blind people as it just scrolls you down to the content, what is done by tabbing anyways..

File: theme/open/views/registration/forms/einwilligungserklaerung.hbs (hunk 1 above)
Comment (luizzaly) [reply theme/open/views/registration/forms/einwilligungserklaerung.hbs:24]: the whole file must be translated! Students are working on translating the whole platform and I guess this is not done yet...As our main focus is a11y I did not implement i18 keys here...

Comment (haniapalac) [reply theme/open/views/registration/forms/einwilligungserklaerung.hbs:24]: you can concatenate warnings and labels in an aria-label, then a SR should read both

**Pull request merged**